### PR TITLE
feat(bsd) add logs for scanning and configuration

### PR DESCRIPTION
**Pull request by carolinemodic**
Adds logs to BSD when configuring an election, scanning, deleting a batch, and exporting the CVR.

### CODE REVIEW

File: frontends/bsd/src/screens/ballot_eject_screen.tsx [hunk 1]
```diff
@@ -71,12 +73,24 @@ function doNothing() {
   console.log('disabled'); // eslint-disable-line no-console
 }
 
+const SHEET_ADJUDICATION_ERRORS = [
```
Comment (eventualbuddha) [inline frontends/bsd/src/screens/ballot_eject_screen.tsx:76]:
```suggestion
const SHEET_ADJUDICATION_ERRORS: Array<PageInterpretation['type']> = [
```

File: libs/logging/src/log_event_ids.ts [hunk 2]
```diff
@@ -440,6 +455,99 @@ const DownloadedScanImageBackup: LogDetails = {
...
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User had initiated configuring the machine from a ballot package. The ballot package will be loaded from the usb, each ballot will be configured, the scanner will be configured, and then the election configuration will be complete.',
+  defaultMessage: 'Loading ballot package from USB and configuring machine...',
+};
+const BallotPackagedLoadedFromUsb: LogDetails = {
+  eventId: LogEventId.BallotPackagedLoadedFromUsb,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'The ballot package has been read from the usb. Success or failure indicated by disposition.',
```
Comment (eventualbuddha) [inline libs/logging/src/log_event_ids.ts:469]:
```suggestion
    'The ballot package has been read from the USB drive. Success or failure indicated by disposition.',
```

File: libs/logging/src/log_event_ids.ts [hunk 3]
```diff
@@ -440,6 +455,99 @@ const DownloadedScanImageBackup: LogDetails = {
     'User downloaded a backup file of the scanned ballot image files and CVRs. Success or failure indicated by disposition.',
 };
 
+const ConfigureFromBallotPackageInit: LogDetails = {
+  eventId: LogEventId.ConfigureFromBallotPackageInit,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'User had initiated configuring the machine from a ballot package. The ballot package will be loaded from the usb, each ballot will be configured, the scanner will be configured, and then the election configuration will be complete.',
```
Comment (eventualbuddha) [inline libs/logging/src/log_event_ids.ts:462]:
```suggestion
    'User had initiated configuring the machine from a ballot package. The ballot package will be loaded from the USB drive, each ballot will be configured, the scanner will be configured, and then the election configuration will be complete.',
```

File: libs/logging/src/log_event_ids.ts [hunk 4]
```diff
@@ -440,6 +455,99 @@ const DownloadedScanImageBackup: LogDetails = {
...
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'Scanning continued by user after errors and/or warning stopped scanning. Log will indicate if the sheet was tabulated with warnings, or if the user indicated removing the ballot in order to continue scanning.',
+};
+const ScanAdjudicationInfo: LogDetails = {
+  eventId: LogEventId.ScanAdjudicationInfo,
+  eventType: LogEventType.ApplicationStatus,
+  documentationMessage:
+    'Information about a ballot sheet that needs adjudication from the user. The possible unresovables errors are InvalidTestModePage when a test mode ballot is seen when scanning in live mode or vice versa, InvalidElectionHashPage when a sheet for the wrong election is seen, InvalidPrecinctPage when a sheet for an invalid precinct is seen, UninterpretedHmpbPage for a HMPB ballot that could not be read properly, UnreadablePage for a sheet that is unrecognizable as either a HMPB or BMD ballot, and BlankPage for a blank sheet. Warnings that the user can choose to tabulate with a ballot include MarginalMark, Overvote, Undervote, WriteIn, UnmarkedWriteIn, and BlankBallot (a ballot where there are no votes for any contest).',
```
Comment (eventualbuddha) [inline libs/logging/src/log_event_ids.ts:542]:
```suggestion
    'Information about a ballot sheet that needs adjudication from the user. The possible unresovable errors are InvalidTestModePage when a test mode ballot is seen when scanning in live mode or vice versa, InvalidElectionHashPage when a sheet for the wrong election is seen, InvalidPrecinctPage when a sheet for an invalid precinct is seen, UninterpretedHmpbPage for a HMPB ballot that could not be read properly, UnreadablePage for a sheet that is unrecognizable as either a HMPB or BMD ballot, and BlankPage for a blank sheet. Warnings that the user can choose to tabulate with a ballot include MarginalMark, Overvote, Undervote, WriteIn, UnmarkedWriteIn, and BlankBallot (a ballot where there are no votes for any contest).',
```

File: libs/utils/src/ballot_package.ts [hunk 5]
```diff
@@ -200,7 +200,7 @@ async function readBallotPackageFromFilePointer(
 ): Promise<BallotPackage> {
   assert(window.kiosk);
   const zipFile = await openZip(await window.kiosk.readFile(file.path));
-  return readBallotPackageFromZip(zipFile, file.name, file.size);
+  return await readBallotPackageFromZip(zipFile, file.name, file.size);
```
Comment (eventualbuddha) [inline libs/utils/src/ballot_package.ts:203]: This should be equivalent, but doesn't hurt to leave it. Was there an issue here?

**Review by eventualbuddha — APPROVED** [review]

File: libs/utils/src/ballot_package.ts (hunk 5 above)
Comment (carolinemodic) [reply libs/utils/src/ballot_package.ts:203]: oh I think I just changed it while debugging something and didn't realize it wasn't there before when switching back to the original code

**Pull request merged**